hasher-matcher-actioner/webapp/src/Api.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ */
+
+// TODO figure out root path. For now copy your own URL gateway here
+const API_ROOT = 'https://<APPID>.execute-api.us-east-1.amazonaws.com'
+// TODO replace with real auth someday
+const API_TOKEN = '<TOKEN>'
+
+export async function getAPI(
+    route,
+    params = {},
+  ) {
+    const urlWithParams = new URL(`${API_ROOT}/${route}`);
+    urlWithParams.searchParams.append("access_token", API_TOKEN);
+    for (const [key, value] of Object.entries(params)) {
+      urlWithParams.searchParams.append(key, value);
+    }
+
+    const response = await fetch(urlWithParams.toString(), {
+        method: 'GET',
+      });
+    return response.json();
+  }
+
+export async function postAPI(
```

Comment:
See nit on 'consumer' oriented APIs below.

postAPI and getAPI is too low level. You could make those private and expose a surface of high-level functions that do specific things.

Advantages:
* changing the API contract becomes local to this file
* you can make the APIs typed strictly which further makes it reuseable.